trussworks/react-uswds
# feat: Deprecate default heading levels

# Summary

- Deprecates default headingLevels on a few components. 
- A future PR will mandate the specification of a heading level.
## Related Issues or PRs

towards #1946 

## How To Test

In Storybook, change a component to not specify a heading level. See that a deprecation warning appears in the console.
